internal/envoy/auth/ext_authz_filter.go

```diff
@@ -84,25 +87,36 @@ func NewHTTPExternalAuthorizationFilter(upstreamHostname string, upstreamPort ui
 		}
 	}
 
-	httpService := &envoy_auth_v3.HttpService{
+	httpService := &envoy_extensions_filter_http_ext_authz_v3.HttpService{
 		ServerUri:             serverUri,
 		PathPrefix:            pathPrefix,
 		AuthorizationRequest:  authorizationRequest,
 		AuthorizationResponse: authorizationResponse,
 	}
-	services := &envoy_auth_v3.ExtAuthz_HttpService{
+	services := &envoy_extensions_filter_http_ext_authz_v3.ExtAuthz_HttpService{
 		HttpService: httpService,
 	}
-	authorization := &envoy_auth_v3.ExtAuthz{
-		Services:            services,
-		TransportApiVersion: envoy_config_core_v3.ApiVersion_V3,
+	// withRequestBody := &envoy_extensions_filter_http_ext_authz_v3.BufferSettings{
+	// 	MaxRequestBytes:     10,
+	// 	AllowPartialMessage: true,
+	// 	PackAsBytes:         true,
+	// }
+	authorization := &envoy_extensions_filter_http_ext_authz_v3.ExtAuthz{
+		Services:               services,
+		TransportApiVersion:    TransportApiVersion,
+		IncludePeerCertificate: true,
+		// WithRequestBody:        withRequestBody,
 		StatusOnError: &envoy_type_v3.HttpStatus{
 			Code: envoy_type_v3.StatusCode_ServiceUnavailable,
 		},
+		// // Pretty sure we always want this. Why have an
+		// // external auth service if it is not going to affect
+		// // routing decisions?
+		// ClearRouteCache: false,
```

Comment:
Is this supposed to be commented out?

Reply:
Resolved. Remove commented out code. Thanks.